src/main/java/sirius/biz/process/Processes.java

```diff
@@ -496,6 +496,10 @@ protected boolean updateTitle(String processId, String newTitle) {
 
     /**
      * Adds the given link to the given process.
+     * <p>
+     * When running inside a process the preferred way to add a link is using
```

Comment:
Maybe also add one line to the class javadoc that using the context is preferred to directly using the processes part when accessing/altering the process?